config/locales/en.yml

```diff
@@ -224,6 +224,8 @@ en:
           attributes:
             password_confirmation:
               confirmation: doesn't match with the password entered.
+            photo:
+              spoofed_media_type: file has content that does not match the file type
```

Comment:
Add more text to explain what this means and a user needs to do about this.  (Non-tech people might not read this and then just be stymied about what it really means).  How about:  "The file type (name) does not match what the actual file is.  The file might be misnamed, corrupted, or spoofed.  Please use another file."

Reply:
Changed along suggested approach.